pkg/account/storage/v2/sql/api_key_v2/update_api_key_v2.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE api_key SET
+    name = ?,
+    role = ?,
+    disabled = ?,
+    created_at = ?,
```

Comment:
Can you remove this?
I just noticed that the `created_at` shouldn't be updated.